packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -109,6 +113,53 @@ function clientConfig(env) {
 	};
 }
 
+function getBabelEsmPlugin(config) {
```

Comment:
why did you move this out btw?

Reply:
Oh right.. This was intermittent decision which should have not been moved out

Reply:
oh 👀